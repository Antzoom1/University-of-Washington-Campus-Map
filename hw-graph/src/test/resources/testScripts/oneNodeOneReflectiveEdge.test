# Tests that a singular edge that points to and from one node.

# Create a graph
CreateGraph oneNodeGraph

# Add one node
AddNode oneNodeGraph 1

# Add one edge
AddEdge oneNodeGraph 1 1 e1

# Print the node in the graph and the outgoing edges from 1
ListNodes oneNodeGraph
ListChildren oneNodeGraph 1