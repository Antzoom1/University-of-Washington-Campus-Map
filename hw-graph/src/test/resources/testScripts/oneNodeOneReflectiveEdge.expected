# Tests that a singular edge that points to and from one node.

# Create a graph
created graph oneNodeGraph

# Add one node
added node 1 to oneNodeGraph

# Add one edge
added edge e1 from 1 to 1 in oneNodeGraph

# Print the node in the graph and the outgoing edges from 1
oneNodeGraph contains: 1
the children of 1 in oneNodeGraph are: 1(e1)